src/main/scala/nl.knaw.dans.easy.bagstore/component/FileSystemComponent.scala

```diff
@@ -224,6 +224,8 @@ trait FileSystemComponent extends DebugEnhancedLogging {
             fileId <- id.toFileId
             location <- toRealLocation(fileId)
           } yield (bagDir.toAbsolutePath.resolve(item.path), location)
+        } recoverWith {
+          case nsfe: NoSuchFileException => Failure(new IllegalArgumentException(s"Bag-id found in fetch.txt can not be found in the bag-store: ${ nsfe.getMessage }"))
```

Comment:
The error message is not correct. This was not correctly formulated in the issue. It should be:
```suggestion
          case nsfe: NoSuchFileException => Failure(new IllegalArgumentException(s"Local-file-uri found in fetch.txt can not be found in the bag-store: ${ nsfe.getMessage }"))
```